openhub/tests/models/test_affiliated_committer_model.py

```diff
@@ -0,0 +1,90 @@
+from django.test import TestCase
+
+from openhub.models import (
+    AffiliatedCommitter,
+    MostCommit,
+    MostRecentCommit)
+
+
+class AffiliatedCommitterModelTest(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        # Set up non-modified objects used by all methods
+        most_commits = MostCommit.objects.create(
+            project='test project',
+            commits='76')
+        most_recent_commit = MostRecentCommit.objects.create(
+            project='test project',
+            date='Jun 2018')
+        AffiliatedCommitter.objects.create(
+            name='Test User',
+            level=2,
+            org='coala',
+            most_commits=most_commits,
+            most_recent_commit=most_recent_commit)
+
+    def test_field_label(self):
+        affiliated_committer = AffiliatedCommitter.objects.get(id=1)
+
+        name = affiliated_committer._meta.get_field('name').verbose_name
```

Comment:
What's the point of testing those verbose names? Maybe only test fields whose verbose names are not by default?

Reply:
Yeah, I just wanted to make sure whenever the name of any field change, there should be a change in tests also :P